### PR TITLE
Add logging for inserting post and link

### PR DESCRIPTION
**Pull request by thescriptninja**

### CODE REVIEW

File: c3po/db/metadata.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 from c3po.db.models import (Artist, ArtistGenre, ArtistSong, Genre, Link, Song,
                             SongGenre, User, UserLikes, UserPosts)
 
+import logging
```
Comment (ghostwriternr) [inline c3po/db/metadata.py:12]: Can you quickly update the logs to use [module-level loggers](https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial) (so we can actually see that the logs are coming from `metadata.py`?

Doing so would be to say:
```py
logger = logging.getLogger(__name__)
```
and use `logger.info` to print logs.

**Review by ghostwriternr — CHANGES_REQUESTED** [review]

File: c3po/db/metadata.py (hunk 1 above)
Comment (thescriptninja) [reply c3po/db/metadata.py:12]: Gotcha

**Review by mukul-mehta — APPROVED** [review]

**Review by ghostwriternr — DISMISSED** [review]
Noice

**Review by ghostwriternr — APPROVED** [review]
PEP8 ki jai ho

**Pull request merged**